src/main/kotlin/no/nav/syfo/personstatus/api/v2/model/PersonOversiktStatusDTO.kt

```diff
@@ -24,6 +24,7 @@ data class PersonOversiktStatusDTO(
     val arbeidsuforhetvurdering: ArbeidsuforhetvurderingDTO?,
     val friskmeldingTilArbeidsformidlingFom: LocalDate?,
     val oppfolgingsoppgave: OppfolgingsoppgaveDTO?,
+    val senOppfolgingKandidatUbehandlet: Boolean,
```

Comment:
Hva med å ta inspirasjon fra de over og sende med en DTO? Selv om vi ikke vet om skjemasvar kan det være andre greier vi er interesserte i som vi kanskje vil vise i oversikten? I første omgang så kan vi jo for eks. ta med status på kandidaten 😊
```suggestion
    val senOppfolgingKandidat: SenOppfolgingDTO?,
```

```typescript
data class SenOppfolgingDTO {
    status: Status
}
```